drivers/user_permission_report/app/views/user_permission_report/warehouse_reports/reports/index.haml

```diff
@@ -0,0 +1,60 @@
+:scss
+  .user-permission-report td ul {
+    list-style: none;
+    padding-left: 0;
+  }
+  li {
+    margin-bottom: 0.5em;
+  }
```

Comment:
This should go in `app/assets/stylesheets/application/modules/_warehouse_reports.scss` We try really hard to not embed styles directly in pages to keep that separation.

Reply:
✅